src/Illuminate/Database/Schema/Grammars/ChangeColumn.php

```diff
@@ -121,7 +121,7 @@ protected static function getDoctrineColumnChangeOptions(Fluent $fluent)
             $options['length'] = static::calculateDoctrineTextLength($fluent['type']);
         }
 
-        if ($fluent['type'] === 'json') {
+        if ($fluent['type'] === 'json' || $fluent['type'] === 'binary') {
```

Comment:
Can you please use `in_array()`?

Reply:
I changed it to use `in_array()`.

Reply:
Should `in_array()` be used in strict mode (pass true for third parameter) or was the type checking not needed in the first place?

Reply:
I don't see a case where the strict mode would be necessary, we are always comparing strings.